src/index.js

```diff
@@ -179,12 +180,14 @@ function loaderForModel(model, attribute, attributeField, options = {}) {
     delete findOptions.rejectOnEmpty;
 
     if (findOptions.limit && keys.length > 1) {
+      const limit = findOptions.offset && findOptions.offset > 0 ? [findOptions.limit, findOptions.offset] : findOptions.limit;
```

Comment:
any reason not just to pass offset as an option?

Reply:
It doesn't work. Apparently `groupedLimit` doesn't support an offset parameter. I actually didn't find any docs related to it.

Reply:
But arrays work? I wasn't aware that is official API in sequelize v3 (and after)

Reply:
Array works. I did not test with v3 but it works with v4+